cypress/e2e/signUp.cy.js

```diff
@@ -1,5 +1,21 @@
+import { generateUser } from '../support/generate';
+
 describe('Sign Up page', () => {
   it('should provide an ability to register new account', () => {
+    const user = generateUser();
+    cy.visit('https://react-mobx.realworld.io/#/register');
+
+    cy.get('[placeholder=Username]')
+      .type(user.username);
+
+    cy.get('[placeholder=Email]')
+      .type(user.email);
+
+    cy.get('[placeholder=Password]')
+      .type(user.password);
```

Comment:
These `cy.get()` commands are selecting elements based on their placeholder text. This can make your tests fragile because if the placeholder text changes, the test will break. A better approach would be to use data attributes (like `data-cy` or `data-test`) to select elements. This way, your tests are not reliant on CSS or JS for selecting elements and are less likely to break due to changes in the application's code.